platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,480 @@
+
+###### Examples Scenarios:
+
+An API is provided which abstracts the creation and maintenance of databases and provides users with any information they require to leverage that platform capability such as a connection string, location for secret data, and dashboard with observability data.
+
+##### Level 4, Optimized - Managed services
+
+Platform capabilities are transparently integrated into the tools and processes that teams already use to do their work. Some capabilities are provisioned automatically, such as observability or identity management for a deployed service. When users hit the edges of the provided services, there is an opportunity to move past automated solutions and customize for their needs without leaving the internal offerings because platform capabilities are considered building blocks which are built into the transparent and automatic compositions to meet the higher-level use cases while enabling deeper customization where necessary.
+
+###### Characteristics:
+
+* It is clear what capabilities are differentiating for the organization and which are not, allowing the internal teams to invest in custom solutions only where they can not leverage industry standards.
+* While capabilities are surfaced in a consistent way, there is no one way to use a capability. Some are best suited as CLI tools for use in scripts whereas others benefit from integration into where the user is writing code in their editors and IDEs.
+* The value of individual capabilities is extended with a focus on the flow of both software development and release, leading to a focus on how to combine capabilities into higher level offerings.
+* While capabilities are often provided in packages, super users are enabled to decompose these higher level offerings in order to optimize when and where they need to.
+
+###### Examples Scenarios:
+
+* Observability agents are injected into every workload and an OIDC proxy is placed in front of all applications
+* By default every new project receives a space in a task runner (pipelines) and a runtime environment (K8s namespace), however a project can opt into other options such as serverless runtime.
+* From a catalog in a Service Now portal a user selects "Provision a Database". Automation provisions an RDS database and sends a URL and credentials to the user.
+
+### Operations
+<h4 style="color:gray"><i>How are platforms and their capabilities planned, prioritized, developed and maintained?</i></h4>
+
+Operation of a computing platform means running and supporting its capabilities and their features over their whole lifetime, including acceptance of new requests, initial releases, upgrades and extensions, ongoing maintenance and operations, user support and even deprecation and termination. Organizations and their platform teams choose platforms and capabilities to create and maintain and can prioritize the most valuable and impactful initiatives.
```

Comment:
Curious as to why have we specified `computing platform` and not just `platform` since all across the document we've used platform.
```suggestion
Operation of a computing platform means running and supporting its capabilities and their features over their whole lifetime, including acceptance of new requests, initial releases, upgrades and extensions, ongoing maintenance and operations, user support, and even deprecation and termination. Organizations and their platform teams choose platforms and capabilities to create and maintain and can prioritize the most valuable and impactful initiatives.
```

Reply:
Great point, also removed computing!